templates/credentials/faq.html

```diff
@@ -3,149 +3,229 @@
 {% block title %}Canonical Credentials -- Frequently asked questions{% endblock %}
 
 {% block meta_copydoc %}
-https://docs.google.com/document/d/1S_lSeRPtSqEcoZ9LK0Evn9QAoK1FrA2w40N-WkxYGic/edit
+  https://docs.google.com/document/d/1S_lSeRPtSqEcoZ9LK0Evn9QAoK1FrA2w40N-WkxYGic/edit
 {% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--image is-dark" style="background-image: url(https://assets.ubuntu.com/v1/4a1f9f33-suru-cut.png);
-                    background-position: 100% 60%">
-  <div class="row u-equal-height">
-    <div class="col-6">
-      <h1>CUE.01 Linux FAQ</h1>
+  <section class="p-strip--image is-dark"
```

Comment:
Image strip has been deprecated. For surus, one of suru variants should be used
https://vanillaframework.io/docs/examples/patterns/suru/variants

This one seems to be closest to `p-suru--fan-top is-dark`, although possibly some other tweaks may be needed, as it may not work well with just a heading (due to min height of the background)